rpc/client.go

```diff
@@ -0,0 +1,45 @@
+package rpc
+
+import (
+	"github.com/hashicorp/yamux"
+	log "github.com/sirupsen/logrus"
+	"net"
+	"net/rpc"
+	"net/rpc/jsonrpc"
+)
+
+type Client struct {
+	*rpc.Client
+}
+
+func NewClient() *Client {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewClient should have comment or be unexported